src/components/tabs/demoStretchHeight/index.html

```diff
@@ -0,0 +1,35 @@
+<div ng-controller="DemoStretchHeightCtrl" ng-cloak>
+  <md-block style="height: 460px" layout="column">
```

Comment:
`md-block` should be used as a class. There is no `<md-block>` element.

Reply:
@Splaktar addressed.